crates/ide-completion/src/completions.rs

```diff
@@ -357,6 +357,15 @@ impl Completions {
         variant: hir::Variant,
         local_name: Option<hir::Name>,
     ) {
+        if let PathCompletionCtx {
+            kind: PathKind::Pat { pat_ctx }, use_tree_parent: false, ..
```

Comment:
This actually works without `use_tree_parent: false`, just for explicit and extra safety.